### ISSUE TITLE
loader and fallbackLoader are deprecated

### ISSUE THREAD

**Issue by jc275** [vuejs-templates/webpack#480]
Running e2e test triggers warnings:
```
fallbackLoader option has been deprecated - replace with "fallback"
loader option has been deprecated - replace with "use"
```

Warning goes away if [build/utils.js](https://github.com/vuejs-templates/webpack/blob/master/template/build/utils.js#L31) is updated replace
```js
      return ExtractTextPlugin.extract({
        loader: sourceLoader,
        fallbackLoader: 'vue-style-loader'
      })

```
with
```js
      return ExtractTextPlugin.extract({
        use: sourceLoader,
        fallback: 'vue-style-loader'
      })

```

**Comment by cschweda** [comment]
Yep. Same errors. Same manual fix.

**Comment by kazupon** [comment]
Fixed with 0b8051f

**Comment by faithyang** [comment]
I modify fallbackLoader to fallback，there is no first warning.
